src/fu-engine.c

```diff
@@ -3764,7 +3764,7 @@ fu_engine_add_releases_for_device_component (FuEngine *self,
 	return TRUE;
 }
 
-static GPtrArray *
+GPtrArray *
```

Comment:
This should have some gtk-doc if it's being exported now

Reply:
But it does not get publicly exported, I do not see any of the `fu-engine.h` methods in `Fwupd-2.0.gir`. `fu-engine.h` is also not installed to the system.

Reply:
At least elsewhere in fu-engine.h we've been adding gtk-doc in case that stuff exports later IIRC? @hughsie ?

Reply:
Nth, no hard requirement. I'm not sure if bother here.